lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -183,6 +183,8 @@ class LighthouseViewerReport {
       this._isNewReport = true;
 
       this.replaceReportHTML(JSON.parse(str));
+
+      ga('send', 'event', 'report', 'upload');
```

Comment:
`upload` is a confusing choice of word in this function :) Which upload is this? Just for drag and drop on the page?